baseplate/integration/pyramid.py

```diff
@@ -107,13 +127,25 @@ class BaseplateConfigurator(object):
 
     """
 
-    # TODO: remove the default on trust_trace_headers once apps are updated
-    def __init__(self, baseplate, trust_trace_headers=False,
-                 edge_context_factory=None):
+    def __init__(self, baseplate, trust_trace_headers=None,
+                 edge_context_factory=None, header_trust_handler=None):
         self.baseplate = baseplate
-        self.trust_trace_headers = trust_trace_headers
+        self.trust_trace_headers = bool(trust_trace_headers)
```

Comment:
Maybe we should change this to `self._trust_trace_headers` and add:

```
@property
def trust_trace_headers(self):
    warn_deprecated('don't do this')
    return self._trust_trace_headers
```

And then at some point we could either remove it or have it raise a special exception.